app_envs.go

```diff
@@ -0,0 +1,45 @@
+package api
+
+import (
+	"encoding/json"
+	"gopkg.in/nullstone-io/go-api-client.v0/types"
+	"net/http"
+	"path"
+)
+
+type AppEnvs struct {
+	Client *Client
+}
+
+func (e AppEnvs) Get(appName, envName string) (*types.AppEnv, error) {
+	res, err := e.Client.Do(http.MethodGet, path.Join("apps", appName, "envs", envName), nil, nil, nil)
```

Comment:
One thing I realized when working on my current card is that apps can not be referenced uniquely by org_name and app_name. You either have to include the stack_name as a third argument or use the app_id for uniqueness.

Reply:
That's kinda weird. 
The endpoint for apps is `/orgs/:org_name/apps/:name` and it's still dependent on uniqueness within a single stack?

I think an app should be unique across orgs.
It's nice when the CLI can just say `nullstone deploy <app> <env>` -- you don't need to know the stack.

Reply:
Updated to use `appId` instead of `appName`.